bigbluebutton-html5/imports/api/users/server/handlers/validateAuthToken.js

```diff
@@ -45,7 +45,8 @@ export default function handleValidateAuthToken({ payload }) {
         addWelcomeChatMessage(meetingId, userId);
       }
 
-      return Logger.info(`Validated auth token as '${validStatus}' user=${userId} meeting=${meetingId}`);
+      return Logger.info(`Validated auth token as '${validStatus}'
+        user=${userId} meeting=${meetingId}`);
```

Comment:
So it formats the message after all..
Then it's up to @antobinary if we want to see log messages in this format.